schema/testing.proto

```diff
@@ -238,4 +230,128 @@ message Test {
 
   // A description of the test.
   string description = 3;
+
+  // A blueprint is an explicit definition of methods and requests that are needed
+  // to be made to test this specific test case. Ideally this would be represented
+  // by something more robust like CEL, but as of writing this, I am unsure if CEL
+  // is ready.
+  message Blueprint {
+    // The id of this blueprint.
+    string id = 1;
```

Comment:
What are examples of this value? (I expect this should be called something like `code`, since ID is overloaded.)

Reply:
Added a resource path and renamed to `name`.